frontend/packages/modelServing/src/extensions.ts

```diff
@@ -3,18 +3,20 @@ import type { ProjectDetailsTab } from '@odh-dashboard/plugin-core/extension-poi
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { SupportedArea } from '@odh-dashboard/internal/concepts/areas/types';
 
-const extensions: ProjectDetailsTab[] = [
+const projectDetailsTabs: ProjectDetailsTab[] = [
   {
     type: 'app.project-details/tab',
     properties: {
       id: 'model-server', // same value as ProjectSectionID.MODEL_SERVER
       title: 'Models',
-      component: () => import('./components/ModelsProjectDetailsTab'),
+      component: () => import('./components/projectDetails/ModelsProjectDetailsTab'),
     },
     flags: {
       required: [SupportedArea.PLUGIN_MODEL_SERVING],
     },
-  },
+  } satisfies ProjectDetailsTab,
 ];
 
+const extensions: ProjectDetailsTab[] = [...projectDetailsTabs];
```

Comment:
Not sure if you need a type here -- but if you're going to do it, maybe `typeof variable` so you don't have to update both spots when things change.